routes/favorites.js

```diff
@@ -6,11 +6,12 @@ module.exports = (db) => {
   // show a user's favorite list
   router.get("/", (req, res) => {
    let queryString = `
-    SELECT favorites.*, items.title, items.price, items.description
+    SELECT DISTINCT favorites.item_id, items.title, items.price, items.description, items.is_available
```

Comment:
So we're not displaying the multiple favourites, but our database is storing multiple favourites. It works, but a person could spam the button and fill our database with junk data! Perhaps as an improvement the additional favourite could not be posted if that favourite already exists in the database?

Reply:
I thought about it too but not sure how to implement it

Reply:
Yeah... I'm kind of drawing a blank, too! If you'd like a collaborator to be your rubber ducky and pair program with, I'd be happy to take a swing at making it even shinier together. That said, the bug's fixed and the demo's almost ready. I'll go ahead and merge now!